RecoTracker/TkHitPairs/interface/HitPairGeneratorFromLayerPair.h

```diff
@@ -28,6 +28,15 @@ class HitPairGeneratorFromLayerPair {
 
   void hitPairs( const TrackingRegion& reg, OrderedHitPairs & prs,
                  const edm::Event & ev,  const edm::EventSetup& es, Layers layers);
+  static void doublets2(
```

Comment:
I would call this function also 'doublets'.
Purely a taste thing. 
Only change if Tracking convenors confirm.